src/content/docs/en/core-concepts/routing.mdx

```diff
@@ -219,6 +220,79 @@ const { title, text } = page;
 </html>
 ```
 
+## Redirects
+
+Sometimes you will need to redirect your readers to a new page, either permanently because your site structure has changed, or in response to an action such as logging in to an authrenticated route.
```

Comment:
```suggestion
Sometimes you will need to redirect your readers to a new page, either permanently because your site structure has changed, or in response to an action such as logging in to an authenticated route.
```